clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -117,7 +117,7 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
     QualType SrcT = C->getSubExpr()->getType();
     QualType DstT = C->getType();
     if (!isCastSafe(DstT, SrcT) && !Info.hasPersistentConstraints(C, Context)) {
-      auto CVs = CB.getExprConstraintVars(C->getSubExpr());
+      auto CVs = CB.getExprConstraintVarsSet(C->getSubExpr());
```

Comment:
Possible conflict location again; John is doing a bunch of stuff around inserting casts.